docs/fxa-engineering/release-process.md

```diff
@@ -181,11 +171,21 @@ Git is a flexible tool and there are other options if other scenarios arise.  Do
 
 ![Diagram showing a security release](assets/fxa-release3.png)
 
-A security release will make use of the `fxa-private` repository.  This diagram illustates a regular release happened as usual, but halfway through the sprint an important security patch needed to be pushed live.  The patch, in red above, lands on the current train branch of `fxa-private`.  Once it's been reviewed and passes tests, `release.sh patch` is run to generate the changelogs as usual.  Operations is involved at this point to deploy to production directly from `fxa-private` so the patch is never seen before it's live.
+A security release will make use of the `fxa-private` repository.  This diagram illustates an important security patch being pushed live midway through sprint 100.  Firstly, we need to bring `fxa-private` up to speed since it probably hasn't been used in a while.
+
+```bash
+git clone git@github.com:mozilla/fxa-private.git
+cd fxa-private
+git remote add fxa https://github.com/mozilla/fxa.git
+git fetch fxa --tags
+git merge v0.100.0   # Replace this with whatever tag is currently live
+```
 
-Once the fix is verified in production, the patch is cherry-picked back for `fxa`.
+Next, make a patch for whatever needs fixing and pull request against `fxa-private`.  The patch is reviewed as normal and lands on `fxa-private`.
 
+Operations is involved at this point to deploy to production directly from `fxa-private` so the patch is never seen before it's live.
```

Comment:
@jrgm @jbuck I'm not sure how you deploy from fxa-private right now.  If you need this to be tagged I'll add that to these docs.